data/src/main/java/com/teamagam/gimelgimel/data/message/adapters/MessageDataMapper.java

```diff
@@ -197,20 +216,25 @@ private URL tryParseUrl(String remoteUrl) {
             return null;
         }
 
-        private Alert convertAlertData(AlertData content, String id) {
+        private GeoAlert convertGeoAlertData(AlertData content, String id) {
             AlertEntity entity = mGeoEntityDataMapper.transformIntoAlertEntity(
                     id,
                     content.source,
                     content.location,
                     content.severity);
-            return new Alert(content.source,
+            return new GeoAlert(content.source,
```

Comment:
The order of the parameters is opposite to their order at Alert's constructor. Why is that?